docs/development_suite/api-console/api-design/listeners.md

```diff
@@ -0,0 +1,76 @@
+---
+id: listeners
+title: Create Listeners
+sidebar_label: Create Listeners
+---
+
+## What is a Listener
+
+A Listener is a logical entity configured on your API Gateway that checks for incoming connection requests directed to your cluster.  
+Each listener is mapped to a single port and defines rules to route incoming requests from external clients to a resource in your cluster, such as a specific microservice, exposed through endpoints.
+
+This page describes the features of a Listener and how to configure these from the **Listeners** section of the [Design area](/development_suite/api-console/api-design/overview.md) of the Console.
+
+:::info
+Currently Listeners are available only to Projects configured to use an **Envoy API Gateway**.
+:::
+
+## Create a Listener
+
+Follow this three steps in order to create the new Listener and make an Endpoint reachable from outside the Console:
+- create the new Listener resource
+- expose the Endpoint on the Listener
+- define a new Traefik IngressRoute for the newly exposed API Gateway port
+
+### Defining the Listener 
+
+Creating a new Listener is as easy as clicking on the **Add listener** button and specifying its `name` and `port`. Both these two fields must be unique on your Project.  
+
+Optionally, you can define whether to:
+- **expose new endpoints on the listener** by deafult
+- **expose all existing endpoints** on the listener
```

Comment:
For these two options, I would explain a little more extensively what they imply